app/src/js/components/Table/Metadata.js

```diff
@@ -15,8 +15,10 @@ class Metadata extends React.Component {
             value = meta[2](value);
           }
           return acc.concat([
-            <dt key={`meta-${meta[1]}--dt`}>{meta[0]}</dt>,
-            <dd key={`meta-${meta[1]}--dd`}>{value}</dd>
+            <div className="meta__row">
+              <dt key={`meta-${meta[1]}--dt`}>{meta[0]}</dt>
+              <dd key={`meta-${meta[1]}--dd`}>{value}</dd>
+            </div>
```

Comment:
Looks like this change is causing a test failure in `test/components/reconciliation-reports/reconciliation-report.js`. In the test "show individual report", we have

```
  // ReconciliationReport is configured to use 6 metaAccessors,
  // so there will be 6 groups of dt, dd elements for a total of 12
  t.is(MetadataWrapperChildren.length, 12);
```

Since the dt and dd elements are now grouped, we should be able to do 
`  t.is(MetadataWrapperChildren.length, 6);` instead.

Reply:
Thanks for helping with the test failure. I notices it's failing in reports since I added a div group but was trying to figure out how to fix it. Much appreciated.